app/controllers/admin/responses_controller.rb

```diff
@@ -20,19 +21,29 @@ def create
   end
 
   def edit
-    render :edit_legacy
+    render_design_system("edit", "edit_legacy", next_release: true)
```

Comment:
i think this needs to be `next_release: false` since it won't be going out in the next release

Reply:
👍